src/marshmallow/fields.py

```diff
@@ -416,7 +417,13 @@ class Nested(Field):
     default_error_messages = {"type": "Invalid type."}
 
     def __init__(
-        self, nested, *, default=missing_, exclude=tuple(), only=None, **kwargs
+        self,
+        nested: SchemaABC,
+        *,
+        default: typing.Any = missing_,
+        exclude: typing.Union[typing.List, typing.Tuple] = tuple(),
```

Comment:
Can also be a `set`, right? Same with `only`.

Might want to DRY this with `only` and `exclude` in `Schema`.